src/global_distrib/mod_global_distrib_utils.erl

```diff
@@ -43,6 +43,7 @@
 binary_to_metric_atom(Binary) ->
     List = lists:filtermap(fun
                                ($.) -> {true, $_};
+                               ($ ) -> {true, $_};
```

Comment:
I was thinking about it, but I think it would not match - `$ ` is a number and `" "` is a string (a list with one number). An alternative would be just writing 32...
<img width="461" alt="obraz" src="https://user-images.githubusercontent.com/34194983/119957619-83191b80-bfa2-11eb-9c2c-281b95756f00.png">

Reply:
Right, good one 😄